sdk/lib/crt/setjmp/arm/setjmp.s

```diff
@@ -41,6 +41,9 @@
 
     LEAF_ENTRY longjmp
 
+    /* Save 2nd argument */
+    mov r2, r1
```

Comment:
Yeah, but why not use `r2` as temporary and leave `r1` as is, instead?

Reply:
I ain't familiar to arm assembly.